impl/src/main/java/com/okta/sdk/impl/oauth2/AccessTokenRetrieverServiceImpl.java

```diff
@@ -134,17 +134,15 @@ String createSignedJWT() throws InvalidKeyException, IOException {
         PrivateKey privateKey = parsePrivateKey(getPemReader());
         Instant now = Instant.now();
 
-        String jwt = Jwts.builder()
+        return Jwts.builder()
             .setAudience(tokenClientConfiguration.getBaseUrl() + TOKEN_URI)
             .setIssuedAt(Date.from(now))
-            .setExpiration(Date.from(now.plus(1L, ChronoUnit.HOURS)))
+            .setExpiration(Date.from(now.plus(50, ChronoUnit.MINUTES)))
```

Comment:
nit: a code comment on why it is set to `50` mins would be helpful for posterity. @VitaliiTytarenko-okta